fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraSessionImpl.java

```diff
@@ -51,8 +52,10 @@
 public class FedoraSessionImpl implements FedoraSession {
 
     // The default timeout is 3 minutes
+    @VisibleForTesting
     public static final String DEFAULT_TIMEOUT = Long.toString(ofMinutes(3).toMillis());
```

Comment:
This is not actually used in any testing code... and can presently be safely `private`.